qiita_core/util.py

```diff
@@ -93,3 +101,21 @@ def wrapper(*args, **kwargs):
         with TRN:
             return func(*args, **kwargs)
     return wrapper
+
+
+def get_qiita_version():
+    """Returns the Qiita version and Git sha if present"""
+    # the actual repo is the abspath of the current file without
+    # qiita_core
+    current_fp = dirname(__file__)
+    len_for_trimming = len('qiita_core/')
+    git_repo_path = current_fp[:-len_for_trimming]
+
+    try:
+        repo = Repo(git_repo_path)
+        sha = '@' + repo.active_branch.commit.hexsha[0:7]
+    except:
```

Comment:
The reason why you want to only catch a specific exception is so as to
not mask real errors, does that make sense? It is briefly mentioned in
the PEP8 document, look for "When catching exceptions, mention ..." 
https://www.python.org/dev/peps/pep-0008/#exception-names

On (Feb-08-16|13:17), Antonio Gonzalez wrote:
> > @@ -93,3 +101,21 @@ def wrapper(*args, **kwargs):
> >          with TRN:
> >              return func(*args, **kwargs)
> >      return wrapper
> > +
> > +
> > +def get_qiita_version():
> > +    """Returns the Qiita version and Git sha if present"""
> > +    # the actual repo is the abspath of the current file without
> > +    # qiita_core
> > +    current_fp = dirname(__file__)
> > +    len_for_trimming = len('qiita_core/')
> > +    git_repo_path = current_fp[:-len_for_trimming]
> > +
> > +    try:
> > +        repo = Repo(git_repo_path)
> > +        sha = '@' + repo.active_branch.commit.hexsha[0:7]
> > +    except:
> 
> We could but not sure why will we want to do this, could you comment? Basically, not sure which errors should we catch, I think all-exceptions is fine. 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/1640/files#r52218796